### ISSUE TITLE
Typo in SC 3.3.1 Error Identification 

### ISSUE THREAD

**Issue by white-wine** [w3c/wai-wcag-quickref#231]
It says: 
See also 3.3.**1**: Error Suggestion

It should be:
See also 3.3.**3**: Error Suggestion

**Comment by fstrr** [comment]
Thanks for this. I've fixed the typo in the WCAG repo.

**Comment by iadawn** [comment]
Has been fixed
